server/googleActions.js

```diff
@@ -154,13 +155,15 @@ GoogleActions = {
 				resource : { name        : `${brew.title}.txt`,
 										 description : `${brew.description}`,
 										 properties  : { title      : brew.title,
-										 							 	 published  : brew.published,
-																	   lastViewed : brew.lastViewed,
-																	   views      : brew.views,
-																	   version    : brew.version,
-																		 renderer   : brew.renderer,
-																	   tags       : brew.tags,
-																	   systems    : brew.systems.join() }
+										 							    published  : brew.published,
+																	    lastViewed : brew.lastViewed,
+																	    views      : brew.views,
+																	    version    : brew.version,
+																	    renderer   : brew.renderer,
+																	    tags       : brew.tags,
+																	    systems    : brew.systems.join(),
+																	    pageCount  : (brew.text.match(/\\page/g) || []).length + 1
```

Comment:
```suggestion
																	    pageCount  : (brew.text.match(/^\\page/gm) || []).length + 1
```

We've actually got two slightly different ways of counting pages, depending on the renderer. Legacy splits pages *anywhere* there is a `\page`. V3 only splits if the line starts with `\page`.

We should use the V3 version preferably (faster Regex since it knows to only start at the beginning of a line instead of scanning the *entire text*) *or* make sure to handle both cases specifically.

*or*, pass the page count as a `brew` property into the save function instead of trying to calculate it here. Might be cleaner and be a bit more DRY.

**Edit:** yeah, after reading the rest of this, let's do that: pass in pageCount as a brew property.

Reply:
Committed.